mscore/musescore.cpp

```diff
@@ -3649,7 +3649,10 @@ static bool convert(const QString& inFile, const QJsonArray& outFiles, const QSt
       MasterScore* score = mscore->readScore(inFile);
       if (!score)
             return false;
+      mscore->appendScore(score);
       mscore->setCurrentScore(score);
+      mscore->setCurrentView(0, 0);
+      mscore->setCurrentView(1, 0);
```

Comment:
These 2 lines warrant a comment IMHO

Reply:
Why set twice anyway?

Reply:
Exactly, that's what I'd like explained ;-)

Reply:
I'm following the behaviour in the [`loadScores` function](https://github.com/musescore/MuseScore/blob/8eb74bbe41128515b3d1b193ae38ce1001a4adf2/mscore/musescore.cpp#L3394) that uses [this behaviour](https://github.com/musescore/MuseScore/blob/8eb74bbe41128515b3d1b193ae38ce1001a4adf2/mscore/musescore.cpp#L3478). I don't understand how this works either, but if anyone else understands why it's necessary in `loadScores` I'd appreciate an explanation.

Reply:
Something similar is done at the end of `loadScores()`, also without any explanation though, so I still have no idea what it might be good for

Reply:
Okay, I've tried removing them, and I get the following errors: 

```
/home/adam/personal/musescore-aam/libmscore/cmd.cpp:Ms::Score::startCmd: ===startCmd()
file:///home/adam/personal/musescore-aam/install/share/mscore-3.4/plugins/transposer_0.qml:onRun: Hello Transposer
file:///home/adam/personal/musescore-aam/install/share/mscore-3.4/plugins/transposer_0.qml:onRun: Score title: Flying Home (That Solo)
file:///home/adam/personal/musescore-aam/install/share/mscore-3.4/plugins/transposer_0.qml:onRun: Original key: -4
/home/adam/personal/musescore-aam/mscore/musescore.cpp:Ms::MuseScore::cmd: MuseScore::cmd <transpose-up>
/home/adam/personal/musescore-aam/mscore/musescore.cpp:Ms::MuseScore::cmd: invalid state 0002 0000
/home/adam/personal/musescore-aam/mscore/musescore.cpp:Ms::MuseScore::cmd: MuseScore::cmd <transpose-up>
/home/adam/personal/musescore-aam/mscore/musescore.cpp:Ms::MuseScore::cmd: invalid state 0002 0000
/home/adam/personal/musescore-aam/mscore/musescore.cpp:Ms::MuseScore::cmd: MuseScore::cmd <transpose-up>
/home/adam/personal/musescore-aam/mscore/musescore.cpp:Ms::MuseScore::cmd: invalid state 0002 0000
/home/adam/personal/musescore-aam/mscore/musescore.cpp:Ms::MuseScore::cmd: MuseScore::cmd <transpose-down>
/home/adam/personal/musescore-aam/mscore/musescore.cpp:Ms::MuseScore::cmd: invalid state 0002 0000
file:///home/adam/personal/musescore-aam/install/share/mscore-3.4/plugins/transposer_0.qml:onRun: Transposed key: -4
file:///home/adam/personal/musescore-aam/install/share/mscore-3.4/plugins/transposer_0.qml:onRun: Staying Alive
/home/adam/personal/musescore-aam/libmscore/cmd.cpp:Ms::Score::endCmd: ===endCmd() 3
```

I *think* the `setCurrentView` is necessary as it sets the current score correctly so that plugins can be run on it. As I say though, I simply don't understand how it works.

Reply:
In any case, I think an explanation of this is outside the scope of the PR, so I'm resolving this.